client/src/main/java/io/pravega/client/stream/ReaderGroup.java

```diff
@@ -91,9 +113,8 @@
      * were a new reader group.
```

Comment:
The javadoc for `updateConfig` must be changed to reflect that the current position is unaffected, not "exactly as though it were a new reader group".

Reply:
removed the `updateConfig` method for the first cut.